app/controllers/subjects_controller.rb

```diff
@@ -13,7 +13,15 @@ def show
   end
 
   def all
-    @subjects = TreePreloader.new.preload
+    @subjects =
+      begin
+        subjects =
+          if params[:search].present?
+            Subject.where("lower(unaccent(name)) LIKE ?", "%#{I18n.transliterate(params[:search].downcase)}%")
```

Comment:
I think that you can use `lower(unaccent(?))` instead of calling `I18n.transliterate` and `downcase`. This way we make sure it's making the same modifications to the string.

Reply:
Good call!

Reply:
Updated in [bcf11a7](https://github.com/cedarcode/mi_carrera/pull/429/commits/bcf11a70d6cee3a50c6a7266e57c980753c17741)